app/views/layouts/_topbar.haml

```diff
@@ -21,12 +13,8 @@
           %a.dropdown-button{'chartLineData-activates': 'dropdown1', href: "#!"}
             Repositories
             %i.material-icons.right
-      - if login?
-        %li
-          = link_to 'Logout', logout_path, method: :destroy, class: 'right'
+        %li= link_to 'Setup Repositrory', setup_path, class: 'right'
```

Comment:
Setup Reposit~r~ory 🇬🇧 

Please move all texts to translations.

Also, this button should move us to editing current repository or be `Setup repositories`.